src/ophys_etl/modules/segmentation/denoising_utils/averaging.py

```diff
@@ -0,0 +1,36 @@
+import numpy as np
+from scipy.ndimage import uniform_filter1d, gaussian_filter1d
+
+
+def temporal_filter1d(video: np.ndarray,
+                      size: int,
```

Comment:
It looks like you are allowing `size` to be a float in the case of Gaussian filters. Should the type hint be updated?